### PR TITLE
docs: update contributing

### LINKED ISSUE #110: Cannot run `npm run dev` and run example.md
`npm run dev`

```
11:06:16 AM [vite] Internal server error: EMFILE: too many open files, open '/home/jgurhem/Projets/slidev-theme-aneo/slidev/node_modules/katex/dist/katex.min.js'
   (x4)
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: EMFILE: too many open files, open '/home/jgurhem/Projets/slidev-theme-aneo/components.d.ts'] {
  errno: -24,
  code: 'EMFILE',
  syscall: 'open',
  path: '/home/jgurhem/Projets/slidev-theme-aneo/components.d.ts'
}

Node.js v18.12.1
```

Before that I ran `npm ci` and `npm install` as my env was not up to date

### PR DESCRIPTION
**Pull request by Barbapapazes**
fix #110

### CODE REVIEW

**Review by aneojgurhem — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -23,10 +23,20 @@ See the [documentation](https://barbapapazes.github.io/slidev-theme-aneo/) for m
 
 ## Contributing
 
-- `npm install`
-- `npm run dev` to start theme preview of `example.md`
+First, you need to install `pnpm` globally:
+
+```bash
+npm install -g pnpm
```
Comment (aneojgurhem) [inline README.md:29]:
```suggestion
sudo npm install -g pnpm
```

File: README.md [hunk 2]
```diff
@@ -23,10 +23,20 @@ See the [documentation](https://barbapapazes.github.io/slidev-theme-aneo/) for m
...
+_Pnpm is used instead of npm because it is faster and more reliable._ Learn more about [pnpm](https://pnpm.io/).
+
+Then, you can run the following commands:
+
+- `pnpm install`
+- `pnpm run dev` to start theme preview of `example.md`
 - Edit the `example.md` and style to see the changes
-- `npm run export` to generate the preview PDF
-- `npm run screenshot` to generate the preview PNG
+- `pnpm run export` to generate the preview PDF
```
Comment (aneojgurhem) [inline README.md:39]: I had to run `npm i -D playwright-chromium` so it is missing in the package.json.
I still cannot run `pnpm run export`

Comment (Barbapapazes) [reply README.md]: Are you trying to contribute (installing this repo on your machine) or are you to replicate these command for your project?

Comment (aneojgurhem) [reply README.md]: I'm trying to contribute to this repo

**Pull request merged**